javascript/uuidgenerator.js

```diff
@@ -0,0 +1,29 @@
+/* eslint no-console: "error" */
+
+// npm install uuid
+
+/* ----------------------------------- */
+/* ----------------------------------- */
+
+// using version 4 to create uuid
+// generates a random uuid
+
+const { v4: uuidv4 } = require("uuid");
+Console.log(`${uuidv4()}`);
```

Comment:
Codacy found an issue: ['Console' is not defined.](https://app.codacy.com/gh/tasnimzotder/hacktoberfest-2020/pullRequest?prid=6281261)